samples/java-eventsourced-counter/src/test/java/com/example/actions/CounterJournalToTopicActionTest.java

```diff
@@ -24,6 +28,14 @@ public void increaseTest() {
     assertEquals(1, result.getReply().getValue());
   }
 
+  @Test
+  public void increaseMetadataTest() {
+    TestKitActionContext context = new TestKitActionContext(new HashMap<String, String>(Map.of("myKey","myValue")));
```

Comment:
I think this is the wrong approach, Metadata is something that comes with each request, not something global across several calls.

I would have expected the user facing testkit API to add overloads to call methods specific metadata, like this:

```java
testKit.increase(ValueIncreased.newBuilder...build(), Metadata(stuff))
```

Reply:
I see. Yep, now that you mention it, makes a lot of sense. I'll do that.